debian/base/Dockerfile.rpm

```diff
@@ -36,12 +38,13 @@ ENV PYTHON_PIP_VERSION="9.0.1"
 # Confluent
 ENV SCALA_VERSION="2.11"
 ENV CONFLUENT_MAJOR_VERSION="3"
-ENV CONFLUENT_MINOR_VERSION="2"
+ENV CONFLUENT_MINOR_VERSION="3"
 
-ENV CONFLUENT_PATCH_VERSION="1"
+ENV CONFLUENT_PATCH_VERSION="0"
```

Comment:
Wrap building of this Dockerfile in a script that automatically extracts from git branch and sets these as env vars.